data_steward/utils/bq.py

```diff
@@ -68,22 +246,24 @@ def delete_dataset(project_id,
     client.delete_dataset(dataset_id,
                           delete_contents=delete_contents,
                           not_found_ok=not_found_ok)
-    logging.info(f'Deleted dataset {project_id}.{dataset_id}')
+    LOGGER.info('Deleted dataset %s.%s', project_id, dataset_id)
```

Comment:
This is a pylint thing.  string interpolation is preferred to f strings because it is only interpolated when needed.  this makes a difference in projects that heavily use logging.  this project doesn't, but i don't want to introduce or encourage bad coding practices.

Reply:
pylint flags it as a warning.  not an error.